source/boards/data-and-archives.rst

```diff
@@ -0,0 +1,106 @@
+Import your data
+================
+
+You can import data from other tools to use with Boards.
+
+Import from Asana
+-----------------
+
+This node app converts an Asana JSON archive into a ``.focalboard`` archive.
+
+1. Log into your Asana account.
+2. From the Asana Board Menu, select **Export/Print > JSON** to create an archive file (which you'll use in Boards).
+3. Save the file locally, e.g. to ``asana.json``.
+4. In your local copy of Boards, open a terminal window.
+5. Navigate to ``focalboard/webapp``.
+6. Run ``npm install``.
+7. Change directory to ``focalboard/import/asana``.
+8. Run ``npm install``.
+9. Run ``npx ts-node importAsana.ts -i <asana.json> -o archive.focalboard``.
+10. Open Boards and select **Settings > Import archive** and select ``archive.focalboard``.
```

Comment:
Is this correct? If so, why isn't the user selecting the asana JSON file? If the steps above have changed that, we should probably note it here.

Reply:
Yes this is still correct. The above command (step 9)...converts `asana.json` to `archive.focalboard` Essentially all of these `imports` are actually conversions to our import format.  We then import our file format.

Reply:
Prior to Step 10, we probably want to have them open the board in which they want to import into.